### PR TITLE
Create basic tutorial

### LINKED ISSUE #644: Tutorial for new users
We need a way to quickly show new users what to do when they first get on the site.

### PR DESCRIPTION
**Pull request by krazemon**
Fixes #644

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -816,3 +816,33 @@ textarea {
...
+.tutorial-panel {
+  position: fixed;
+
+}
+
+.tutorial-text {
+  margin: 10px 10px 10px 10px;
+}
+
+.feed-explanation {
```
Comment (ritmatter) [inline css/app.css]: Why all the empty classes?

File: js/components/TutorialPanel.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    clazz: React.PropTypes.string,
+  },
+
+  getInitialState: function() {
+    return {
+      step: 0,
+    };
+  },
+
+  goToNext: function() {
```
Comment (ritmatter) [inline js/components/TutorialPanel.jsx]: I think it's sort of weird that we had the tutorial manager, which starts the tutorial and "manages" the tutorial, but it doesn't continue to get the tutorial to the next step. It's more like the thing that initializes the tutorial.

Not saying that holds back this PR, but I think it is kind of confusing.

**Comment by ritmatter** [comment]
LGTM, just check out my comments

File: css/app.css (hunk 1 above)
Comment (krazemon) [inline css/app.css]: To make it clear that each one can be moved around if need be.

File: js/components/TutorialPanel.jsx (hunk 2 above)
Comment (krazemon) [inline js/components/TutorialPanel.jsx]: That's fair. I tried doing that at first, but it didn't make sense or work very well. We could remove the TutorialManager all together. Would you prefer that?

Comment (ritmatter) [inline js/components/TutorialPanel.jsx]: Yea I think that would be better if its not too much work

**Pull request merged**